app/controllers/album.controller.js

```diff
@@ -16,7 +17,7 @@ exports.create = (req, res) => {
   const album = {
     title: req.body.title,
     artistId: req.body.artistId,
-    numberOfTracks: req.body.numberOfTracks
+    numberOfTracks: 0
```

Comment:
Was wondering why this is hardcoded to 0?

Reply:
this is in create album function since we create an album with zero tracks and we start adding tracks when the album exists... if you checkout dev branch for UI you can see that when we add a new album user is not able to enter the number of tracks. any thoughts ?

Reply:
oh sounds good, that makes sense.